src/app/common/table/filter/asy-header-date-filter/asy-header-date-filter.component.ts

```diff
@@ -93,24 +93,24 @@ export class AsyHeaderDateFilterComponent extends AsyAbstractHeaderFilterCompone
 		if (this.enabled) {
 			if (!this.isCustom) {
 				const now = DateTime.now();
+
 				if (this.direction === 'past') {
 					$gte = now.minus({ [this.duration]: this.count });
 					$lte = now;
 				} else {
 					$gte = now;
 					$lte = now.plus({ [this.duration]: this.count });
 				}
-				$gte = $gte.startOf(this.duration);
-				$lte = $lte.endOf(this.duration);
 			} else if (this.customRange?.length === 2) {
 				$gte = DateTime.fromJSDate(this.customRange[0]);
 				$lte = DateTime.fromJSDate(this.customRange[1]);
 			}
-		}
-		// Normalize to start/end of day
-		if (this.isCustom || this.duration !== 'hour') {
-			$gte = $gte.startOf('day');
-			$lte = $lte.endOf('day');
+
+			// Normalize to start/end of day
+			if (this.isCustom || this.duration !== 'hour') {
+				$gte = $gte.startOf('day');
```

Comment:
@jrassa - when we're finding the start and end of the day, is that localized to the user's start and end? Or using some standardized start / end, like UTC?

Reply:
It is standardizing to UTC.

Reply:
Actually, previously it was localized to the user's start/end, but will now be standardized to UTC.

Reply:
Sounds good - thanks!